Java/app/src/main/java/com/taboola/sdk4example/sdk_classic/MetaAdUICustomization.java

```
@@ -0,0 +1,126 @@
+            @Override
+            public boolean onItemClick(String placementName, String itemId, String clickUrl, boolean isOrganic, String customData) {
+                Log.d(TAG, "onItemClick");
+                return super.onItemClick(placementName, itemId, clickUrl, isOrganic, customData);
+            }
+
+            @Override
+            public void onAdReceiveSuccess() {
+                super.onAdReceiveSuccess();
+                Log.d(TAG, "onAdReceiveSuccess");
+            }
+
+            @Override
+            public void onAdReceiveFail(String error) {
+                super.onAdReceiveFail(error);
+                Log.d(TAG, "onAdReceiveFail " + error);
+            }
+        });
+        tblClassicUnit.setUnitExtraProperties(new HashMap<String, String>() {{
+            put(MetaConst.AUDIENCE_NETWORK_PLACEMENT_ID_KEY, MetaConst.AUDIENCE_NETWORK_PLACEMENT_ID);
+        }});
+        Typeface font = loadFont(getActivity(), TYPEFACE_ARIAL_BOLD);
+
+        // Create custom style properties for the branding
+        TBLUiStyleProperties brandingStyleProperties = new TBLTextStylePropertiesBuilder(ELEMENT_TYPE_BRANDING)
+                .setFontLightColor(R.color.design_default_color_error)
+                .setFontDarkColor(R.color.colorPrimary)
+                .setFontSize(TEXT_FONT_SIZE)
+                .setTypeface(font)
+                .build();
+
+        // Create custom style properties for the title
+        TBLUiStyleProperties titleStyleProperties = new TBLTitleStylePropertiesBuilder()
+                .setAmountOfSpaceBetweenLines(AMOUNT_OF_SPACE_BETWEEN_LINES)
+                .setLines(NUMBER_OF_LINES)
+                .setFontLightColor(Color.RED)
+                .setFontSize(TEXT_FONT_SIZE)
+                .setFontDarkColor(Color.BLUE)
+                .setTypeface(font)
+                .build();
+
+        // Create custom style properties for the call to action button
```

@DanielFrTB - Why is this code commented out? As mentioned in my previous comment, show the option to publisher that is NOT the default (meaning, hide the CTA).
Leaving commented out code is not a good practice.

---

@tomer-br-taboola 
Mosh ask to put here a commented code for demonstration